js/src/sdk/models/triggers.ts

```diff
@@ -4,12 +4,20 @@ import { BackendClient } from "./backendClient";
 
 import apiClient from "../client/client";
 
-import { ListTriggersData } from "../client";
+import { z } from "zod";
 import { CEG } from "../utils/error";
 import { TELEMETRY_LOGGER } from "../utils/telemetry";
 import { TELEMETRY_EVENTS } from "../utils/telemetry/events";
 
-type RequiredQuery = ListTriggersData["query"];
+const ZTriggerQuery = z.object({
```

Comment:
Consider adding documentation for the `ZTriggerQuery` schema to explain the purpose of each field. This will help other developers understand the filtering options available.